.github/workflows/deployment-test-template.yml

```
@@ -13,20 +13,22 @@
+      - name: Stop containers
+        run: docker compose down
 
       - name: Start containers
-        run: docker compose -f ${{ inputs.compose_file }} up -d --build
+        run: |
+          echo "${{ secrets.DATABASE_PASSWORD }}" > ./db_password.conf;
+          export DATABASE_PASSWORD_FILE=$PWD/db_password.conf;
+          docker compose -f ${{ inputs.compose_file }} up -d --build
 
       - name: Check if all services are up and running
         run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:6: Quote this to prevent word splitting

(shellcheck)